Lynx/2.9.0_dev6/Lynx/2.9.0_dev5/Resources/Description

```diff
@@ -0,0 +1,5 @@
+[Name] Lynx
+[Summary] Fully-featured, text-based World Wide Web browser.
+[License] GNU General Public License (GPL)
+[Description] Lynx is a fully-featured WWW client for users running cursor-addressable, character-cell display devices (e.g., vt100 terminals, terminal emulators running on PCs or Macs, or any other character-cell display). It will display HTML documents containing links to files on the local system, as well as files on remote systems running HTTP, HTTPS, gopher, FTP, WAIS, NNTP, finger, or cso/ph/qi servers, and services accessible via logins to telnet, tn3270, or rlogin accounts.
+[Homepage] http://lynx.isc.org/
```

Comment:
@ermo I think this file should not exist here?

Reply:
@Nuc1eoN yeah, that looks weird -- it's not present in my tree though...

Reply:
@Nuc1eoN Fixed in 170a57e68c3db49593e4b21521261969bc7ffd4e